### ISSUE TITLE
dfhack portion of PyLNP.json empty

### ISSUE THREAD

**Issue by XVicarious** [Lazy-Newb-Pack/Lazy-Newb-Pack-Linux#9]
This causes none of the preinstalled hacks to show up in the DFHack tab. 

 I'll give this a look and possibly make a pull request myself.

**Comment by lethosor** [comment]
@PeridexisErrant is the file your pack uses available on Github?

**Comment by XVicarious** [comment]
I just looked, he does:

https://github.com/PeridexisErrant/LNP-shared-core/blob/master/LNP/PyLNP.json

Last updated January 30th.  Not sure how much has changed though.

**Comment by lethosor** [comment]
[This](https://github.com/Lazy-Newb-Pack/LNP-shared-core/blob/master/LNP/PyLNP.json) is the file in this organization's repo, although PeridexisErrant's fork is ahead by a few commits. I'll merge that in, but I think there have been changes made in that pack more recently that aren't in either repo.

**Comment by PeridexisErrant** [comment]
Yeah, updating that is on my to-do list.  It's not quite a copy-paste job though, because there's some stuff specific to the windows version or dependant on some config not shared by other packs.  With a new DF release soonish though it just moved up in priority to ensure rebuilding the pack from scratch is reasonably practical.

**Comment by BeauBouchard** [comment]
I previously left this out, because it was difficult to test all the presets when compiling the package on multiple distros. Although its something that i enjoyed with the window's version that the linux version lacks. I am looking into adding this now for the next build as well as upgrading DFhack r3, and the other neglected updates for linux package.

**Comment by lethosor** [comment]
It should be cross-platform, so I don't think differences between distros should affect it either.
